tests/cases/fourslash/completionsIsPossiblyTypeArgumentPosition.ts

```diff
@@ -20,7 +20,10 @@
 ////new callAndConstruct<callAndConstruct</*callAndConstruct*/
 
 for (const marker of test.markers()) {
-    if (marker.data && marker.data.valueOnly) {
+    if (marker.data && marker.data.typeOnly) {
+        verify.completions({ marker, includes: "T", excludes: "x" });
```

Comment:
Sorry for the noob question: does `"T"` and `"x"` stand for "Type" and "Value" respectively? I'm trying to understand a little bit of the code by checking some PR's (yours is pretty cool!)

Reply:
@diguifi They do not intrinsically stand for type and value. In this test they are used as stand-ins  for  type and value, but only before they are defined as such above:
```ts
////const x = 0;
////type T = number;
```